scheduler/src/cook/rest/api.clj

```diff
@@ -1438,7 +1429,7 @@
   (let [jobs (wrap-seq (get-in ctx [:request :query-params :job]))
         instances (wrap-seq (get-in ctx [:request :query-params :instance]))
         allow-partial-results (get-in ctx [:request :query-params :partial])
-        instance-uuid->job-uuid #(instance-uuid->job-uuid (d/db conn) %)
+        instance-uuid->job-uuid #(cached-queries/instance-uuid->job-uuid (d/db conn) %)
```

Comment:
Doesn't this namespace kind of imply that you're using a query that has caching (while this particular function doesn't actually have any caching)?

Reply:
Yeah, I see what you mean. Do you have ideas for what would be a better namespace for the `instance-uuid->job-uuid` function? Keeping it in `cook.rest.api` or `cook.tools` (which we originally wanted to do) resulted in circular dependency issues with `cached-queries`. And since we really want to keep `instance-uuid->job-uuid-cache-lookup` in `cached-queries`, we decided that `cached-queries` was the best namespace for `instance-uuid->job-uuid` as well since it resolves the dependency issues and makes clear the relationship between `instance-uuid->job-uuid-cache-lookup` and `instance-uuid->job-uuid`.

Reply:
I'm not seeing a good namespace that already exists.

Maybe this could be solved with function naming? Like maybe the cached one should be named `instance-uuid->job-uuid` and the one with no caching should be named `instance-uuid->job-uuid-no-caching` (or something similar)?

Reply:
That makes sense-- we could also maybe do something like `instance-uuid->job-uuid-datomic-query` for the `no-caching` call. The calls would then be:

`cached-queries/instance-uuid->job-uuid` when using the cache
`cached-queries/instance-uuid->job-uuid-no-caching` or `cached-queries/instance-uuid->job-uuid-datomic-query` when using datomic.

That makes what both calls do be pretty clear. Let me know if you have a preference, and I'll change it to that in the next revision.

Reply:
Sounds great, I like the `-datomic-query` idea